cpp/ql/test/library-tests/dataflow/dataflow-tests/test.cpp

```diff
@@ -1046,6 +1046,14 @@ void memset_test(char* buf) { // $ ast-def=buf ir-def=*buf
 	sink(*buf); // $ ir MISSING: ast
 }
 
+void *realloc(void *, size_t);
+
+void test_realloc() {
+	int *src = indirect_source();
+	int *dest = (int*)realloc(src, sizeof(int));
+	sink(*dest); // $ ir, MISSING: ast
```

Comment:
Note that `ast` is missing. The same is true for the similarly modeled `memset` just above, so I think this is expected.

Reply:
I'm looking into removing the AST dataflow library.